samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -507,6 +507,7 @@ begin_assessment_msg_latest=If multiple submissions are allowed, answers from pr
 begin_assessment_msg_average=If multiple submissions are allowed, your average score will be recorded.
 begin_assessment_msg_attempt_autosubmit_warn_average=You have already submitted this assessment.  If you start another attempt it will be automatically submitted even if you do not answer all questions.  If you start a new attempt now, your score from this attempt will be averaged with all previous attempts to determine your overall assessment score.  Do you want to begin a new attempt of this assessment?
 begin_assessment_msg_attempt_autosubmit_warn_last=You have already submitted this assessment.  If you start another attempt it will be automatically submitted even if you do not answer all questions.  If you start a new attempt now, your score from this attempt will be recorded as your overall score for this assessment.  Do you want to begin a new attempt of this assessment?
+begin_assessment_msg_warn_tabs=To prevent issues, please use only one browser window and tab when taking this assessment and only take one assessment at a time.
```

Comment:
To prevent issues, please use only one browser window. Opening additional assessments or reviewing older assessments in new tabs will cause data loss.

Reply:
That sounds good to me. @fostersdesign any thoughts here?

Reply:
I'd prefer a hybrid of the two options:

To prevent issues, please use only one browser window and tab when taking this assessment. Opening additional assessments or reviewing older assessments in new tabs will cause data loss.

Reply:
I find 'browser window and tab' awkward .... think we should just use one or the other.

Reply:
Mentioning just the window or just the tab, leaves the other option open for them to use and get burned by. 

I'd like to suggest eliminating the device specifics altogether by focusing on our application with something like:

To prevent issues, open only one assessment at a time. Opening additional assessments or reviewing older assessments at the same time may cause issues with your submission.

Reply:
Actually, I want to make another change: "additional assessments" => "multiple assessments" to cover the current assessment too.

To prevent issues, open only one assessment at a time. Opening multiple assessments or reviewing older assessments at the same time may cause issues with your current submission.

Reply:
```suggestion
begin_assessment_msg_warn_tabs=To prevent issues, open only one assessment at a time. Opening multiple assessments or reviewing older assessments at the same time may cause issues with your current submission.
```